man/systemd.network.xml

```diff
@@ -695,6 +695,57 @@
             which is then configured to use them explicitly.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>HomeAddress=</varname></term>
+          <listitem>
+            <para>A boolen. Designates this address the "home address" as defined in
```

Comment:
*boolean*

Why do you have "A boolean", "Takes a boolean", and "Takes a boolean argument" for different settings? The last one sounds the best, please use the same phrase in all descriptions.